app/packs/stylesheets/decidim/decidim_application.scss

```diff
@@ -0,0 +1,9 @@
+// This is a file that can be overridden by the application in order to override styles
+// Notice that this file is included at the very end of the stylesheets packs to have
+// more priority
+//
+// To override CSS variables or Foundation settings use _decidim-settings.scss
+//
+// By default this is empty.
+@import './decidim_application.scss', './decidim.scss';
```

Comment:
This is not necessary

Reply:
In fact, only you need these files:

![imagen](https://user-images.githubusercontent.com/75725233/197744021-f57c8bd2-5a8f-4089-96ae-b069c6d910f3.png)